src/webview/src/components/fbaCheckbox.js

```diff
@@ -27,6 +34,9 @@ import { triggerRestQueryDetails } from './../util';
 
 import { AttributesContext } from './../App';
 import DataProviderEditDialog from './dataProviderEditDialog';
+import TextFieldEditDialog from './textFieldEditDialog';
+
+let toMarkdown = require("marked");
```

Comment:
import { toMarkdown } from 'marked'

might work as well? (I don't understand the differences but would be more consistent?)

Reply:
Taken over from modules page, https://marked.js.org/ . `import { toMarkdown } from 'marked'` seems not to work out of the box.